packages/cache/RELEASES.md

```diff
@@ -2,4 +2,7 @@
 
 ### 0.1.0
 
-- Initial release
\ No newline at end of file
+- Initial release
+
+### 0.2.0
+- Fixes two issues around using zstd compression algorithm
```

Comment:
```suggestion
- Fixes issues with the zstd compression algorithm on Windows and Ubuntu 16.04 [#469](https://github.com/actions/toolkit/pull/469)
```

We might want to link the PRs in the RELEASES notes to help with discoverability